acceptance/Gemfile

```
@@ -20,14 +18,14 @@ gem "beaker-vagrant", *location_for(ENV['BEAKER_VAGRANT_VERSION'] || "~> 0")
 gem "beaker-vmpooler", *location_for(ENV['BEAKER_VMPOOLER_VERSION'] || "~> 1")
 gem "beaker-gke", *location_for(ENV['BEAKER_GKE_VERSION'] || "~> 0.0.3")
 gem 'rake'
-gem 'scooter', '~> 3.2.17'
+gem 'scooter', '~> 4'
```

```suggestion
gem 'scooter', '~> 4.0'
```
So that we don't install 5.0 when it comes out?

---

Good catch, thank you!